commands/Moderation/mute.js

```diff
@@ -0,0 +1,36 @@
+exports.run = async (client, msg, [user]) => {
+  const { muteRole } = msg.guild.settings;
+  if (!msg.guild.settings.muteRole) return msg.reply("Sorry you have not created a mute role for this server.");
```

Comment:
If you have defined muteRole from msg.guild.settings, why are you getting it again?